routes/web.php

```diff
@@ -15,22 +15,5 @@
 |
 */
 
-Route::group([
-    'prefix' => 'cities',
-    'controller' => CityController::class
-], function () {
-    Route::get('/', 'index');
-    Route::post('/', 'store');
-    Route::put('/{city}', 'update');
-    Route::delete('/{city}', 'destroy');
-});
-
-Route::group([
-    'prefix' => 'airlines',
-    'controller' => AirlineController::class
-], function () {
-    Route::get('/', 'index');
-    Route::post('/', 'store');
-    Route::put('/{airline}', 'update');
-    Route::delete('/{airline}', 'destroy');
-});
+Route::get('cities', [CityController::class, 'indexView']);
+Route::get('airlines', [AirlineController::class, 'indexView']);
```

Comment:
```suggestion
Route::get('cities', [CityController::class, 'index']);
Route::get('airlines', [AirlineController::class, 'index']);
```
If you do what I proposed in the comment on `api.php`, you can rename `indexView` to `index` in both of these controllers.